clients/drcachesim/tracer/tracer.cpp

```diff
@@ -856,6 +850,27 @@ create_buffer(per_thread_t *data)
     }
 }
 
+static size_t
+get_v2p_buffer_size()
+{
+    // The handoff interface requites we use dr_raw_mem_alloc and thus page alignment.
+    // We use one page which is 256 entries for 4K pages which is enough for
+    // most cases.
+    return dr_page_size();
+}
+
+static void
+create_v2p_buffer(per_thread_t *data)
+{
+    // The handoff interface requites we use dr_raw_mem_alloc and thus page alignment.
+    data->v2p_buf = (byte *)dr_raw_mem_alloc(get_v2p_buffer_size(),
+                                             DR_MEMPROT_READ | DR_MEMPROT_WRITE, NULL);
+    /* For file_ops_func.handoff_buf we have to handle failure as OOM is not unlikely. */
+    if (data->v2p_buf == NULL) {
+        FATAL("Out of memory for virtual-to-physical buffers.\n");
```

Comment:
Not sure I understand: the allocation failed; do you mean it might be the mmap rlimit and not really no underlying memory?  I would still call that "out of memory" however the allocator failed.

Reply:
The documentation of `dr_raw_mem_alloc` doesn't say that an error always means we're out of memory. But it may be the case that it _usually_ means an OOM. So i thought it'll be better to say "Probable OOM".

Reply:
"Probable" implies we don't know whether there's a problem at all though.  I'll change it to "Allocation of virtual-to-physical buffer failed."